Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -372,10 +376,12 @@ public void ShowRoundEndScoreboard(string text = "")
                     PlayerOOCName = contentPlayerData?.Name ?? "(IMPOSSIBLE: REGISTERED MIND WITH NO OWNER)",
                     // Character name takes precedence over current entity name
                     PlayerICName = playerIcName,
+                    PlayerGuid = userId ?? null,
                     PlayerNetEntity = GetNetEntity(entity),
                     Role = antag
                         ? roles.First(role => role.Antagonist).Name
                         : roles.FirstOrDefault().Name ?? Loc.GetString("game-ticker-unknown-role"),
+                    RolePrototypes = string.Join(", ", roles.Select(role => role.Prototype)),
```

Comment:
Store this as an actual array instead of concatenating it into a string.

Reply:
Also you need to distinguish between job and antag role prototype IDs here, as your `Prototype` field here stores both.